test.sh

```diff
@@ -20,7 +20,7 @@ fi
 
 FAIL=0
 
-if ! ./gradlew cleanTest test --no-build-cache; then
+if ! ./gradlew cleanTest integrationTest --no-build-cache; then
```

Comment:
since `gradlew check` runs both unit test and checkstyle report, i had to move `StudioTest` to a different 'task' as I dont know how to set up Xfvb in gradle itself

Reply:
Is there a different target than ```check``` that only runs checkstyle?
An alternative would be to run gradlew check from the script that launches xvfb. We only depend on the action that runs gradle to simplify the setup but we could always lift the code from there and modify it.

Reply:
I guess we could, but I find it _cleaner_ this way

Reply:
Let's go with this for now.

The PR is already quite large as it is.